### PR TITLE
Feature/tracker documentation

### PR DESCRIPTION
**Pull request by RomanSek**
Preview: https://developers.piwik.pro/en/feature-tracker-documentation/data_collection/index.html

### CODE REVIEW

File: data_collection/web/guides.rst [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+Tracking downloads with inline Javascript
+"""""""""""""""""""""""""""""""""""""""""
+
+There is another alternative for above methods. You can track a download with inline JavaScript code. Insert inline code to HTML tag with ``onclick`` attribute:
+
+.. code-block:: html
+
+  <a href="https://piwik.pro/document-url" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/document-url', 'download']);">Download document</a>
+
+.. hide::
```
Comment (RomanSek) [inline data_collection/web/guides.rst:548]: This directive doesn't exist. What was intention of adding it?

File: data_collection/web/guides.rst [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+Once the goal is defined, every time a tracked event fits its definition, an additional conversion event will be created. We call this procedure an "automatic conversion".
+
+Alternatively, you can trigger a goal manually with the used of
+
+.. code-block:: javascript
+
+    // force conversion of the goal with ID 17
+    _paq.push(["trackGoal", 17]);
+
+.. note::
```
Comment (RomanSek) [inline data_collection/web/guides.rst:712]: This note doesn't contain any text. Should we remove it?

Comment (adaqus) [reply data_collection/web/guides.rst:712]: We can remove it.

File: data_collection/web/guides.rst (hunk 1 above)
Comment (adaqus) [reply data_collection/web/guides.rst:548]: It may be added by some editor. Whatever the reason was, let's just remove it. I will push the change.

Comment (adaqus) [reply data_collection/web/guides.rst]: Done

File: data_collection/web/guides.rst (hunk 2 above)
Comment (adaqus) [reply data_collection/web/guides.rst]: Done

**Pull request merged**